theorems/T000176.md

```diff
@@ -3,9 +3,7 @@ uid: T000176
 if:
   P000075: true
 then:
-  and:
-  - P000073: true
-  - P000016: true
+  P000073: true
 refs:
 - mr: MR1077251
```

Comment:
Not a problem on this line, but the last line should have "{{mr:...}}" to show up in the browser.